codegen_glibmm/templates/stub.cpp.templ

```diff
@@ -23,11 +23,6 @@ guint {{ class_name_with_namespace }}::register_object(
     const Glib::RefPtr<Gio::DBus::Connection> &connection,
     const Glib::ustring &object_path)
 {
-    if (m_registeredObjectId != 0) {
-        g_warning("Cannot register the same object (%s) twice", object_path.c_str());
-        return 0;
-    }
-
     try {
         introspection_data = Gio::DBus::NodeInfo::create_for_xml(interfaceXml0);
```

Comment:
Now that we have "designed more" for register_object() being called multiple times, should we perhaps create introspection_data only once (add `if (!introspection_data)`)?

Reply:
Oops! Good point!